frontend/src/components/ShootTickets/TicketComment.vue

```diff
@@ -82,6 +82,8 @@ export default {
   @import '~vuetify/src/styles/styles.sass';
 
   $cyan-darken-1: map-get($cyan, 'darken-1');
+  $gh-code-background-color: map-get($grey, 'lighten-4');
+  $gh-code-color: map-get($grey, 'darken-4');
```

Comment:
for my taste it could be a bit darker, it's almost not noticable
<img width="1244" alt="Screenshot 2020-11-16 at 21 59 41" src="https://user-images.githubusercontent.com/5526658/99307649-1553a080-2857-11eb-9f32-e7fb457f81c4.png">

Reply:
we agreed on to leave it as is for now